src/sentry/static/sentry/app/components/group/externalIssuesList.tsx

```diff
@@ -24,6 +11,20 @@ import {
   SentryAppInstallation,
   GroupIntegration,
 } from 'app/types';
+import {t} from 'app/locale';
+import AlertLink from 'app/components/alertLink';
+import AsyncComponent from 'app/components/asyncComponent';
+import ErrorBoundary from 'app/components/errorBoundary';
+import ExternalIssueActions from 'app/components/group/externalIssueActions';
+import ExternalIssueStore from 'app/stores/externalIssueStore';
+import IssueSyncListElement from 'app/components/issueSyncListElement';
+import PluginActions from 'app/components/group/pluginActions';
+import SentryAppComponentsStore from 'app/stores/sentryAppComponentsStore';
+import SentryAppExternalIssueActions from 'app/components/group/sentryAppExternalIssueActions';
+import SentryAppInstallationStore from 'app/stores/sentryAppInstallationsStore';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import withOrganization from 'app/utils/withOrganization';
```

Comment:
Did you get your import plugin working again?

Reply:
Yeah, is yours broken?